docs/core/diagnostics/built-in-metrics-diagnostics.md

```diff
@@ -0,0 +1,126 @@
+---
+title: .NET extensions metrics
+description: Review the metrics available for diagnostic .NET extensions libraries.
+ms.topic: reference
+ms.date: 11/02/2023
+---
+
+# .NET extensions metrics
+
+This article describes the metrics built-in for diagnostic .NET extensions libraries produced using the
+<xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the older [EventCounters](event-counters.md) API, see [here](available-counters.md).
+
+- [Meter: `Microsoft.Extensions.Diagnostics.HealthChecks`](#meter-microsoftextensionsdiagnosticshealthchecks)
+  - [Instrument: `dotnet.health_check.reports`](#instrument-dotnethealth_checkreports)
+  - [Instrument: `dotnet.health_check.unhealthy_checks`](#instrument-dotnethealth_checkunhealthy_checks)
+- [Meter: `Microsoft.Extensions.Diagnostics.ResourceMonitoring`](#meter-microsoftextensionsdiagnosticsresourcemonitoring)
+  - [Instrument: `process.cpu.utilization`](#instrument-processcpuutilization)
+  - [Instrument: `dotnet.process.memory.virtual.utilization`](#instrument-dotnetprocessmemoryvirtualutilization)
+  - [Instrument: `system.network.connections`](#instrument-systemnetworkconnections)
+
+## Meter: `Microsoft.Extensions.Diagnostics.HealthChecks`
+
+### Instrument: `dotnet.health_check.reports`
+
+| Name | Instrument Type | Unit (UCUM) | Description |
+| ---- | --------------- | ----------- | ----------- |
+| `dotnet.health_check.reports` | Counter | `{report}` | Number of times a health report reported the health status of an application. |
+
+| Attribute | Type | Description | Examples | Presence |
+|---|---|---|---|---|
+| `dotnet.health_check.status` | string | The health status of an application. | `Healthy`; `Unhealthy` | Always |
+
+`dotnet.health_check.status` is one of the following:
+
+| Value | Description |
+|---|---|
+| `Degraded` | An application was in degraded state. |
+| `Healthy` | An application was healthy. |
+| `Unhealthy` | An application was unhealthy. |
+
+Available starting in: .NET extensions 8.0.
```

Comment:
This moniker doesn't really correspond to anything in the API ref docs. Perhaps use ".NET Platform Extensions 8" instead? Or just ".NET 8"?